metricshub-doc/src/site/markdown/prometheus/grafana.md

```diff
@@ -0,0 +1,49 @@
+keywords: grafana, dashboard, green, power consumption, carbon, CO₂
+description: How to import the Hardware (MetricsHub) dashboards for Grafana.
+
+# Grafana Dashboards
+
+<!-- MACRO{toc|fromDepth=1|toDepth=2|id=toc} -->
+
+**MetricsHub** collects the hardware health and performance metrics as well as the sustainability KPIs of any server, storage system, or network switch available in your on-prem infrastructure and exposes the telemetry data in the **Hardware Main (MetricsHub)**, **Hardware Site (MetricsHub)**, and **Hardware Host (MetricsHub)** Grafana dashboards.
+
+![MetricsHub Main - Overview of all monitored sites](../images/grafana-metricshub-main.png)
+
+![MetricsHub Site - Metrics associated to one site and its monitored hosts](../images/grafana-metricshub-site.png)
+
+![MetricsHub Host - Metrics associated to one host and its internal components](../images/grafana-metricshub-host.png)
+
+In this section, you will learn how to import the **MetricsHub** dashboards in Grafana. 
+
+## Prerequisites
+
+Before importing the **MetricsHub** dashboards in Grafana, ensure that you have:
+
+1. [configured the monitoring of your resources](../configuration/configure-monitoring.md)
+2. [configured the sustainability metrics](../guides/configure-sustainability-metrics.md)
+3. [configured the Prometheus Server](prometheus.md)
+4. run both **MetricsHub** and the **Prometheus server**.
+
+## Importing the MetricsHub dashboards
+
+1. Log on to Grafana
+2. [Import](https://grafana.com/docs/grafana/latest/dashboards/manage-dashboards/#export-and-import-dashboards) the dashboard(s)
```

Comment:
I think, the Grafana doc has been moved, we should update the link to `[Import](https://grafana.com/docs/grafana/latest/dashboards/build-dashboards/import-dashboards/)`